src/osd/ClassHandler.h

```diff
@@ -110,10 +109,13 @@ class ClassHandler
       const std::string& list);
 
 public:
+  Mutex mutex;
```

Comment:
can we lock the mutex in `add_embedded_class()`, so we don't need to expose it as a public member variable?